rotkehlchen/db/evmtx.py

```diff
@@ -484,6 +488,14 @@ def delete_transactions(
             tx_hashes=tx_hashes,
             chain_id=chain_id,  # type: ignore[arg-type] # comes from SUPPORTED_EVM_CHAINS
         )
+        dbevents.delete_genesis_tx_events(write_cursor, address)
```

Comment:
Can you add a comment saying that this is needed for the case where multiple addresses tracked were involved in the genesis tx?